docs/ConsistencyPreference.md

```diff
@@ -0,0 +1,17 @@
+
+
+# ConsistencyPreference
+
+## Enum
+
+
+* `UNSPECIFIED` (value: `"UNSPECIFIED"`)
+
+* `MINIMIZE_LATENCY` (value: `"MINIMIZE_LATENCY"`)
+
+* `HIGHER_CONSISTENCY` (value: `"HIGHER_CONSISTENCY"`)
+
+* `UNKNOWN_DEFAULT_OPEN_API` (value: `"unknown_default_open_api"`)
```

Comment:
❓ Can we exclude this?

Reply:
I think if we specify `enumUnknownDefaultCase=false` it will be excluded. That would mean if a new value is sent from the server (which is not the case currently) there would be a runtime exception with that setting. 

For consistency it shouldn't be an issue, but it's a global configuration so any enum values sent from the server that don't have a current value would cause an issue, so I'd recommend not setting it